core/api.py

```diff
@@ -171,3 +171,35 @@ def get_student_summary(self, byu_id):
             return parsed_summary
         else:
             return None
+
+    def get_student_enrollments(self, net_id, yearterm):
+        url = (
+            secret_settings.API_STUDENT_ENROLLMENTS_URL
+            + "?net_id="
+            + net_id
+            + "&year_term="
+            + yearterm
+        )
+        headers = {"Authorization": self.build_auth_header()}
+        records_request = requests.get(url, headers=headers)
+        records_json_res = records_request.json()
+        records_data = records_json_res["data"]
+        if len(records_data) > 0:
```

Comment:
This is just a style issue, but I think it's cleaner to just say `if records_data:` since that will always be the same result for any sequence (list, str, tuple, etc.) as `if len(records_data) > 0:`.